plonk-core/src/constraint_system/variable.rs

```diff
@@ -10,7 +10,7 @@
 
 /// The value is a reference to the actual value that was added to the
 /// constraint system
-#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, Default)]
```

Comment:
I don't think it makes sense for the user to be able to construct a `Variable`. If we allow `Default`, we can't enforce this. What's the specific place we need `Default`?

Reply:
I used it here: 
https://github.com/ZK-Garage/plonk/pull/118/files#diff-7580c65a30bf9cd380a7b494e25c0fd4b5760734af32c97a485d9e1d9b52b1f8R534-R535

but you are right. It can be changed for the 0 Variable for example.